tests/test_utils.py

```diff
@@ -93,37 +93,37 @@ def test_get_license(self):
         ret = get_license('Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)')
         self.assertEqual(ret, 'Apache-2.0')
         ret = get_license('Apache')
-        self.assertEqual(ret, 'Apache-1.0')
+        self.assertEqual(ret, 'Apache')
         ret = get_license('BSD-3')
-        self.assertEqual(ret, 'BSD')
-        ret = get_license('Apache-2')
+        self.assertEqual(ret, 'BSD-3-Clause')
+        ret = get_license('Apache2')
         self.assertEqual(ret, 'Apache-2.0')
         ret = get_license('CreativeCommons-Attribution-NonCommercial-NoDerivatives-4.0')
         self.assertEqual(ret, 'CC-BY-NC-ND-4.0')
         ret = get_license('CC BY-NC-SA 4.0')
         self.assertEqual(ret, 'CC-BY-NC-SA-4.0')
-        ret = get_license('BoostSoftwareLicense Version1.0')
```

Comment:
While some of my mappings are definitely incorrect (good catch), the inputs to the `get_license` call were usually inspired by actual package xml strings I was failing to parse correctly.

Please let leave the `get_license` as they are (and correct the mappings as appropriate -- again, very good catch).

Feel free to add the changes you made as new tests, though! For something like this, I'm more than happy to do as many as possible.

Reply:
My mappings are inspired by the values currently included in all package.xml files listed in any distribution in rosdistro and these removed entries don't appear there anymore.